app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError, VaccineError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+
+    if masks_to_buy == 0:
+        return f"Friends can go to {cafe.name}"
+    else:
+        return f"Friends should buy {masks_to_buy} masks"
```

Comment:
The `else` block is not necessary here. You can directly return the message about buying masks without using `else`, as the previous `if` block already returns a value if the condition is met.